node-installer/internal/constants/constants.go

```diff
@@ -14,6 +14,9 @@ import (
 )
 
 var (
+	// Version value is injected at build time.
+	Version = "0.0.0-dev"
```

Comment:
We already inject the version in `internal/constants/constants.go`. Why do we need to inject it here again?

Reply:
Because Contrast imports the node-installer. We could place all versioning in the node-installer

Reply:
Is there a reason the node-installer can't just use the existing values? Either way it seems silly to inject the version in two different places if we could just use one.

Reply:
If the node-installer imports from contrast, we have a cycle (which isn't allowed)

I'm fine with only specifying the version in the node-installer. @katexochen do you want to decide?